### ISSUE TITLE
it should also show Shift, Alt and Ctrl Buttons alone. 

### ISSUE THREAD

**Issue by thearslanstudio** [Code52/carnac#237]
I mean when I press and hold Ctrl or shift or alt it should show on screen, any chance to add this feature?

**Comment by vrnico** [comment]
agreed, trying to use this to screencast Maya commands, but w/o alt, shift, ctrl and mouse bindings it's pretty useless

**Comment by bfritscher** [comment]
Hi, you can try my work while the pull request is not merged https://github.com/bfritscher/carnac/releases

**Comment by NaoBilly** [comment]
> Hi, you can try my work while the pull request is not merged https://github.com/bfritscher/carnac/releases

I love the new version <3 Thanks !
